features/manage-position/Create.tsx

```diff
@@ -141,7 +141,7 @@ const Create = () => {
     const maxTokensToCreate = _gcr > 0 ? collateral / _gcr - startingTokens : 0;
     // Unlike the min collateral, we're ok if we round down the tokens slightly as round down
     // can only increase the position's CR and maintain it above the GCR constraint.
-    setTokens((maxTokensToCreate - 0.0001).toFixed(4));
+    setTokens(Math.floor(maxTokensToCreate).toString());
   };
 
```

Comment:
Similarly, if you FLOOR the tokens to create, the GCR will always be below your desired ratio